src/components/sections/ProductGallery/useOrderedFacets.ts

```diff
@@ -9,15 +9,7 @@ export const useOrderedFacets = (data?: ProductGalleryQueryQuery) => {
 
   return useMemo(() => {
```

Comment:
Will we still need this whole file considering the ordering has been moved to the server?

Reply:
I think this file is important for the sake of avoiding the Filter component blinking after rerendering. But considering we now have the skeletons, let me test it again to double-check it.

Reply:
Yeah, We need this `hook` to avoid reloading all the `Filter` and its `Skeleton` every time the user toggle some facet.
But I think we can remove the `useTotalCount` and change a little the `loading` prop criteria from the Sort/totalCount. I'm going to deal with it on another PR.